spec/support/acceptance/helpers.rb

```diff
@@ -80,6 +81,47 @@ module LitmusHelpers
 end
 
 module Helpers
+  # params is json
+  def curl_get_records(table, query)
+    endpoint = "/api/now/table/#{table}"
+    query ||= {}.to_json
+    query = JSON.parse(query)
+
+    query = query.map do |name, value|
+      "#{CGI.escape(name.to_s)}=#{CGI.escape(value.to_s)}"
+    end
+
+    query = query.join('&')
+    endpt_with_query = query.length > 1 ? "#{endpoint}?#{query}" : endpoint
+    full_result = server.run_shell("curl -v -i --insecure -u mock_user:mock_password -X GET https://localhost:8000#{endpt_with_query}")
+    full_result = full_result.stdout.split("\n")
+    JSON.parse(full_result[full_result.length - 1])['result']
+  end
+  module_function :curl_get_records
+
+  def get_incident_records(params)
+    table = params['table']
+    query = params['url_params']
+
+    query ||= {}.to_json
+    query = JSON.parse(query)
+
+    query = query.map do |name, value|
+      "#{CGI.escape(name.to_s)}=#{CGI.escape(value.to_s)}"
+    end
+
+    query = query.join('&')
+
+    endpoint = "/api/now/table/#{table}"
+    endpt_with_query = query.length > 1 ? "#{endpoint}?#{query}" : endpoint
+
+    full_result = server.run_shell("curl -v -i --insecure -u mock_user:mock_password -X GET https://localhost:8000#{endpt_with_query}")
+    full_result = full_result.stdout.split("\n")
+
+    JSON.parse(full_result[full_result.length - 1])['result']
```

Comment:
Isn't this a nearly exact duplicate of 87-98 above? Couldn't that function above be turned into a generic function that this one could call?